gravitee-apim-console-webui/src/management/api/details/api-portal-details.component.html

```diff
@@ -0,0 +1,178 @@
+    <div class="details-card__header">
+      <div class="details-card__header__info-inputs">
+        <div class="details-card__header__info-inputs__first-row">
+          <mat-form-field class="details-card__header__info-inputs__first-row__name-field" appearance="fill">
+            <mat-label>Name</mat-label>
+            <input #apiName matInput formControlName="name" required />
+          </mat-form-field>
+
+          <mat-form-field class="details-card__header__info-inputs__first-row__version-field" appearance="fill">
+            <mat-label>Version</mat-label>
+            <input matInput formControlName="version" required />
+          </mat-form-field>
+        </div>
+        <div class="details-card__header__info-inputs__second-row">
+          <mat-form-field class="details-card__header__info-inputs__second-row__description-field" appearance="fill">
+            <mat-label>Description</mat-label>
+            <input matInput formControlName="description" required />
+          </mat-form-field>
+
+          <mat-form-field class="etails-card__header__info-inputs__second-row__labels-field" appearance="fill">
+            <mat-label>Labels</mat-label>
+            <gio-form-tags-input
+              formControlName="labels"
+              [addOnBlur]="false"
+              [autocompleteOptions]="labelsAutocompleteOptions"
+            ></gio-form-tags-input>
+          </mat-form-field>
+
+          <mat-form-field class="etails-card__header__info-inputs__second-row__categories-field" appearance="fill">
+            <mat-label>Categories</mat-label>
+            <mat-select formControlName="categories" multiple>
+              <mat-option *ngFor="let category of apiCategories" [value]="category.key"
+                >{{ category.name }} <em *ngIf="category.description"> - {{ category.description }}</em></mat-option
+              >
+            </mat-select>
+          </mat-form-field>
+        </div>
+      </div>
+      <div class="details-card__header__right-coll">
+        <div class="details-card__header__right-coll__media">
+          <gio-form-file-picker class="details-card__header__right-coll__media__picture" formControlName="picture" accept="image/*">
+            <gio-form-file-picker-add-button class="details-card__header__right-coll__media__picture__btn">
+              <span class="details-card__header__right-coll__media__picture__btn__text">
+                Change picture
+                <br />
+                Drag your image
+              </span>
+              <gio-avatar
+                class="details-card__header__right-coll_media__picture__btn__default-avatar"
+                size="114"
+                [name]="apiName.value"
+              ></gio-avatar>
+            </gio-form-file-picker-add-button>
+          </gio-form-file-picker>
+          <gio-form-file-picker class="details-card__header__right-coll__media__background" formControlName="background" accept="image/*">
```

Comment:
The form is dirty when we click on the file picker even if I click on cancel directly. So we have the save bar even if there is no changes. Maybe this is something we can think about. Not here but in a near future. We can try to improve the file picker 🤷

Reply:
i will see later if i can fix this. fist try to finish this page ...